src/Logging/Logging.EventSource/src/EventSourceLogger.cs

```diff
@@ -220,7 +220,7 @@ private string ToJson(IReadOnlyList<KeyValuePair<string, string>> keyValues)
 
             writer.Flush();
 
-            return Encoding.UTF8.GetString(arrayBufferWriter.WrittenMemory.ToArray());
+            return Encoding.UTF8.GetString(stream.ToArray());
```

Comment:
nit: Performance, `stream.TryGetBuffer` and use that to get the Stream, this allocates 3 times instead of twice.